test/integ/test_easy_logging.py

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (c) 2012-2023 Snowflake Computing Inc. All rights reserved.
+#
+
+from __future__ import annotations
+
+import os
+from logging import getLogger
+from pathlib import Path
+from test.integ.conftest import create_connection
+
+import pytest
+
+pytestmark = pytest.mark.skipolddriver
+try:
+    import tomlkit
```

Comment:
Newer Pythons come with [tomllib](https://docs.python.org/3/library/tomllib.html) so we should not add an unnecessary dependency on `tomlkit`.  E.g. the code should read something like:

```python
try:
    from tomllib import dumps
except ModuleNotFoundError:
    from tomlkit import dumps
```

Reply:
but this only exists in 3.11 right? We still have to use tomlkit if we want to support older pythons?

Reply:
Yes, but we should only be conditionally depending on `tomlkit; python < 3.11` so that we don't pull in that extra dependency when we don't need it.  After chatting in Slack and looking, that's a different issue which should be fixed independently, perhaps when we refactor the config manager into its own library.  For now, we'll leave this as is.